src/Umbraco.Community.Contentment/DataEditors/DataList/DataSources/JsonDataListSource.cs

```diff
@@ -24,13 +29,18 @@ public JsonDataListSource(ILogger logger)
             _logger = logger;
         }
 
-        public string Name => "JSON";
+        public string Name => "JSON Data";
 
-        public string Description => "Configure the data source to use JSON data.";
+        public string Description => "Configure the data source (file or url) to use JSON data. Data retrieved from urls are set to UTF-8 encoding.";
 
         public string Icon => "icon-brackets";
 
-        public Dictionary<string, object> DefaultValues => default;
+        public Dictionary<string, object> DefaultValues => new Dictionary<string, object>
+        {
+            { "itemsJsonPath", "$..book[?(@.enabled == true)]" },
```

Comment:
Let's amend this to a more generic example - would `$.*` work?

Reply:
I wanted to include an example which uses filter expressions, and the book-example refers to the same example that I link to for more information. That's why I chose that.

Reply:
OK, let's go with it 👍